components/Navbars/TopBar.tsx

```diff
@@ -1,7 +1,199 @@
+                    <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
+                      <g>
+                        <g>
+                          <path
+                            fill="#8D9290"
+                            d="M12 16.8c-.7 0-1.4-.27-1.93-.8L3.55 9.48a.754.754 0 010-1.06c.29-.29.77-.29 1.06 0l6.52 6.52c.48.48 1.26.48 1.74 0l6.52-6.52c.29-.29.77-.29 1.06 0 .29.29.29.77 0 1.06L13.93 16c-.53.53-1.23.8-1.93.8z"
+                          ></path>
+                        </g>
+                      </g>
+                    </svg>
+                  </div>
+                </div>
+              </label>
+            </div>
+          </div>
+          {/* Action Buttons */}
+          <div className="w-[267px] h-16 p-2 justify-center items-center gap-4 lg:flex-row flex flex-col mt-20  lg:mt-0">
+            <div className=" flex flex-col lg:hidden gap-5 ">
+              <div className="group h flex flex-col ali justify-center items-center gap-1">
+                <Link className={activeLink('/')} href={'/'}>
+                  Explore
+                </Link>
+                {router.pathname === '/' ? <div className="w-6 h-0.5 bg-emerald-600 rounded-lg" /> : null}
+              </div>
+              <div className=" group flex flex-col ali justify-center items-center gap-1 ">
+                <Link className={activeLink('/marketplace')} href={'/'}>
+                  Marketplace
+                </Link>
+                {router.pathname === '/marketplace' ? <div className="w-6 h-0.5 bg-emerald-600 rounded-lg" /> : null}
+              </div>
+            </div>
+            <div className="w-6 h-6 justify-center items-center flex  gap-2">
+              <div className="w-6 h-6">
+                <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" fill="none" viewBox="0 0 24 24">
+                  <g>
+                    <g
+                      stroke="#5B5F5E"
+                      strokeLinecap="round"
+                      strokeLinejoin="round"
+                      strokeMiterlimit="10"
+                      strokeWidth="1.5"
+                    >
+                      <path d="M2 2h1.74c1.08 0 1.93.93 1.84 2l-.83 9.96a2.796 2.796 0 002.79 3.03h10.65c1.44 0 2.7-1.18 2.81-2.61l.54-7.5c.12-1.66-1.14-3.01-2.81-3.01H5.82"></path>
+                      <path d="M16.25 22a1.25 1.25 0 100-2.5 1.25 1.25 0 000 2.5z"></path>
+                      <path d="M8.25 22a1.25 1.25 0 100-2.5 1.25 1.25 0 000 2.5z"></path>
+                      <path d="M9 8h12"></path>
+                    </g>
+                  </g>
+                </svg>
+              </div>
+              <span className=" lg:hidden">Cart</span>
+            </div>
+            <div className="justify-center items-center lg:w-auto w-[100%] gap-2 lg:flex-row flex flex-col">
+              <Button className="rounded-lg border-0 bg-green-50 bg-opacity-50" intent={'secondary'} size={'md'}>
```

Comment:
This is the button Component Added

Reply:
I understand. Please pull from the dev branch one more time. I added the vscode to the gitignore, it should nit be there either.

Reply:
Okay will do that